avaframe/in3Utils/cfgUtils.py

```diff
@@ -93,11 +94,15 @@ def getModuleConfig(module, fileOverride='', modInfo=False, toPrint=True, onlyDe
         log.error(message)
         raise TypeError(message)
 
-    # get path of module
-    modPath = pathlib.Path(module.__file__).resolve().parent
-
-    # get filename of module
-    modName = str(pathlib.Path(module.__file__).stem)
+    if isinstance(module, pathlib.Path):
+        modPath = module.parent
+        # get filename of module
+        modName = module.stem
+    else:
+        # get path of module
+        modPath = pathlib.Path(module.__file__).resolve().parent
+        # get filename of module
+        modName = str(pathlib.Path(module.__file__).stem)
```

Comment:
Since this is also in another function -> maybe make a callable function out of this...

Reply:
moved to a function